### ISSUE TITLE
Camelyon16 Patching

### ISSUE THREAD

**Issue by bryanwong17** [miccaiif/DGMIL#4]
Could you please let me know how to patch the camelyon16 dataset with the same settings as yours?

Could you also let me know how to get 130 slides for testing? When I tried downloading the camelyon16 dataset, I only got 129 slides

**Comment by bryanwong17** [comment]
Hi @miccaiif, for extracting CAMELYON16 patches for 5x magnification, did you follow the default settings on the DSMIL GitHub page?

python .\deepzoom_tiler.py -m 0 -b 5

**Comment by HardworkingLittlequ** [comment]
Hello! Thanks for your attention! You can use DSMIL Github page to generate patches.

**Comment by HardworkingLittlequ** [comment]
Compared with DSMIL, for considerations of computational efficiency and resources, we used 5x (vs. DSMIL 20x) in our experiments. We used a patch size of 512 (vs DSMIL 224), and a patch is labeled as positive if it contains 25% or more cancer areas (not specified in DSMIL). These different settings may result in the difference between the metrics reported by us and those reported by DSMIL.
